### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by mohsenuss91**

### CODE REVIEW

File: config/functions/cron.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+module.exports = {
+  /**
+   * Simple example.
+   * Every monday at 1am.
+   */
+  // '0 1 * * 1': () => {
+  //
+  // }
+};
```
Comment (chinzou) [inline config/functions/cron.js:21]: Why we have a `cron` config in this service?

File: config/database.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module.exports = ({ env }) => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'bookshelf',
+      settings: {
+        client: 'sqlite',
```
Comment (chinzou) [inline config/database.js:7]: can we replace this with SQL like (MySQL, PostgreSQL) or NoSQL services "database should be hosted/managed independently"

File: package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "strapi-plugin-content-manager": "3.0.4",
+    "strapi-plugin-users-permissions": "3.0.4",
+    "strapi-plugin-email": "3.0.4",
+    "strapi-plugin-upload": "3.0.4",
+    "strapi-connector-bookshelf": "3.0.4",
+    "knex": "<0.20.0",
+    "sqlite3": "latest"
+  },
+  "author": {
+    "name": "A Strapi developer"
```
Comment (chinzou) [inline package.json:27]: can we set this to OpenEMP?
can you please delete the file favicon.ico :)

File: config/database.js (hunk 2 above)
Comment (mohsenuss91) [reply config/database.js:7]: @chinzou yes, check this up: https://www.c-sharpcorner.com/article/connecting-to-mysql-database-in-strapi/

File: config/functions/cron.js (hunk 1 above)
Comment (mohsenuss91) [reply config/functions/cron.js:21]: @chinzou  default STRAPI file.

File: extensions/users-permissions/config/jwt.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  jwtSecret: process.env.JWT_SECRET || 'f65c362c-8a6e-441e-9bef-619b026b2bd6'
```
[the remark this answers is not in the document]
Comment (mohsenuss91) [reply extensions/users-permissions/config/jwt.js:2]: @chinzou Yes, I guess!

File: api/messages/models/messages.settings.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "name": "Messages"
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true
+  },
+  "attributes": {
+    "uuid": {
+      "type": "uid",
+      "private": true
```
[the remark this answers is not in the document]
Comment (mohsenuss91) [reply api/messages/models/messages.settings.json:14]: @chinzou  private means the attributes are only available for the members of the class not for the outside of the class.

File: config/functions/cron.js (hunk 1 above)
Comment (chinzou) [reply config/functions/cron.js:21]: if it's safe to delete it, then we should do so as long as we're not using it, to keep our code clean :)

File: config/database.js (hunk 2 above)
Comment (chinzou) [reply config/database.js:7]: that would be way better, messages are too much to be handled with SQLite in production

**Pull request merged**